### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by Mephisto090** [cefsharp/CefSharp.Dom#23] [Ideas]
Do you think it would make sense to translate all errors into German? Otherwise I would just sit down and do it.

**Comment by amaitland** [comment]
I only speak English so translating into a different language isn't practical.

**Reply by amaitland** [reply]
If your keen on translation then I'd suggest translating https://github.com/hardkoded/puppeteer-sharp/ first then those changes can be pulled into this repository.
